### PR TITLE
Show conflicts in entity feed

### PR DESCRIPTION
**Pull request by matthew-white**
Closes getodk/central#531. Closes getodk/central#532. Closes getodk/central-frontend#816.

### CODE REVIEW

File: test/components/entity/diff/table.spec.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  });
+};
+
+describe('EntityDiffTable', () => {
+  describe('Comparing row', () => {
+    const createConflict = () => {
+      const users = Array.from(testData.extendedUsers
+        .createPast(1, { displayName: 'Alice' })
+        .createPast(1, { displayName: 'Bob' })
+        .createPast(1, { displayName: 'David' }));
```
Comment (matthew-white) [inline test/components/entity/diff/table.spec.js:39]: Specify the email and use `sorted()`.

File: test/components/entity/diff/table.spec.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      testData.extendedEntities.createPast(1, { creator: users[0] });
+      testData.extendedEntityVersions
+        .createPast(1, { baseVersion: 1, creator: users[1] })
```
Comment (matthew-white) [inline test/components/entity/diff/table.spec.js:42]:
```suggestion
        .createPast(1, { creator: users[1] })
```

File: test/components/entity/diff/table.spec.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  it('passes correct props to EntityDiffRow for a label change', () => {
+    testData.extendedEntities.createPast(1, { label: 'dogwood' });
+    testData.extendedEntityVersions.createPast(1, { label: 'Dogwood' });
+    const table = mountComponent();
+    const props = table.getComponent(EntityDiffRow).props();
+    props.oldVersion.version.should.equal(1);
+    props.name.should.equal('label');
+  });
+
+  it('passes correct props to EntityDiffRow for a property change', () => {
```
Comment (matthew-white) [inline test/components/entity/diff/table.spec.js:99]: Remove this test? There's not a difference between labels and properties in this component.

File: test/components/entity/diff/row.spec.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+      testData.extendedEntityVersions.createPast(1, { label: 'Dogwood' });
+      const row = mountComponent({
+        props: { name: 'label' }
+      });
+      row.get('td:nth-child(2)').text().should.equal('dogwood');
+    });
+
+    it('shows the old value of a property', () => {
+      testData.extendedEntities.createPast(1, {
+        data: { height: '111111' }
```
Comment (matthew-white) [inline test/components/entity/diff/row.spec.js:67]: Shorten these values.

**Review by matthew-white — COMMENTED** [review]
Notes from interactive code review

**Review by ktuite — APPROVED** [review]
Looking great!

**Pull request merged**